client/my-sites/email/google-workspace-set-up-thank-you/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { useTranslate } from 'i18n-calypso';
+import ThankYouV2 from 'calypso/components/thank-you-v2';
+import { getGoogleMailServiceFamily } from 'calypso/lib/gsuite';
+import { ThankYouGoogleWorkspaceProduct } from 'calypso/my-sites/checkout/checkout-thank-you/redesign-v2/products/google-workspace-product';
+import { useSelector } from 'calypso/state';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import type { ReceiptPurchase } from 'calypso/state/receipts/types';
+
+type GoogleWorkspaceSetUpThankYouProps = {
+	purchase: ReceiptPurchase;
+};
+
+export const GoogleWorkspaceSetUpThankYou = ( { purchase }: GoogleWorkspaceSetUpThankYouProps ) => {
+	const domainName = purchase.meta;
+	const productFamily = getGoogleMailServiceFamily( purchase?.productSlug );
+	const selectedSite = useSelector( getSelectedSite );
+	const translate = useTranslate();
+
+	const footerDetails = [
+		{
+			key: 'footer-extra-info',
+			title: translate( 'Everything you need to know' ),
+			description: translate( 'Explore our support guides and find an answer to every question.' ),
+			buttonText: translate( 'Explore support resources' ),
+			buttonHref: '/support/',
+		},
+		{
+			key: 'footer-plugins',
+			title: translate( "There's a plugin for that" ),
+			description: translate(
+				"With 54,000+ plugins and apps, you'll never outgrow your website. If you can think of it, there's a plugin to make it happen."
+			),
+			buttonText: translate( 'Discover plugins' ),
+			buttonHref: '/plugins/',
```

Comment:
Hmm.... I see that this is the copy we have in from the deck. Although it seems weird that we're promoting plugins after an email purchase instead of showing the email support blurb. Let's use the following instead (it's the same copy from the Titan congrats page, but I updated the support link to point to GWS):

```suggestion
			key: 'footer-email',
			title: translate( "Email questions? We have the answers" ),
			description: translate(
				"Explore our comprehensive support guides and find solutions to all your email-related questions."
			),
			buttonText: translate( 'Email support resources' ),
			buttonHref: '/support/add-email/adding-google-workspace-to-your-site/',
```